src/server/server_family.cc

```diff
@@ -870,6 +878,10 @@ void PrintPrometheusMetrics(const Metrics& m, StringResponse* resp) {
     LOG_FIRST_N(ERROR, 10) << "Error fetching /proc/self/status stats. error "
                            << sdata_res.error().message();
   }
+  append_if_exists("connections.total_bytes", "memory_connections_bytes");
+  append_if_exists("replication.connections.total_bytes", "memory_replication_bytes");
+  append_if_exists("serialization", "memory_serialization_bytes");
+  append_if_exists("unaccounted", "memory_unaccounted_bytes");
```

Comment:
I would expose data_bytes and rss instead of unaccounted, than you can play with the graphs you show and create a graph for 
rss - (connection + replication + seralization + data)

This way you dont need to compute the unaccounted GetMemoryStats

Reply:
you just added connection+replication+serialization, why are you saying that you will need to add them as well?
I think that exposing raw data is better, you can diside on some convention of string that will be exposed in metrixz f.e all stats that match *.memory_bytes , than you can create a graph which does unaccounted = rss - memory*bytes

Reply:
append_if_exists can lead to mistake as if you for some reason deside to change the name in GetMemoryStats response you will need to remember to do the change here as well. 
 exposing unaccounted only if rss > used can also lead to mistakes, as if you have some memory that you will not expose in metricz the user will not know how you computed it and why we have this diff, maybe we are calculating something wrong